StringOperations.java

```diff
@@ -136,11 +120,108 @@ public static void main(String[] args) {
             } else {
                 System.out.print("Would you like to try again? (y/n): ");
             }
-        } while (scanner.nextLine().equalsIgnoreCase("yes"));
+        } while (scanner.nextLine().equalsIgnoreCase("y"));
         System.out.println("Program terminated.");
         scanner.close();
     }
+
+    // Split
+    public static String[] split(String inputString, String pattern) {
+        return inputString.split(pattern);
+    }
+
+    // Max Repeat
```

Comment:
The initial value of max is set to -1, which is fine. However, if the input string is empty, the function will still return a space character (' '). Consider checking if inputString is empty before proceeding and returning an appropriate value or message.